lib/cc/cli/analyze.rb

```diff
@@ -20,6 +20,9 @@ def run
           runner.run
         end
 
+        unless formatter.empty?
+          exit 1
```

Comment:
I agree. Looking forward to when Builder just invokes `codeclimate` as a (containerized) process, the exit status will be part of the interface, and a specific status for issues found makes sense.